components/StatsModal.tsx

```diff
@@ -64,34 +68,9 @@ export default function StatsModal(props: Props) {
   }
 
   function handleShareToTwitter() {
-    const num = Math.ceil(
-      (new Date(date).getTime() - new Date("2022-01-20").getTime()) /
-        24 /
-        60 /
-        60 /
-        1000
-    );
-    const score =
-      gameState.answers[gameState.attempt - 1] === answer
-        ? gameState.attempt
-        : "X";
-    let text = `Katla ${num} ${score}/6%0A%0A`;
-
-    gameState.answers.filter(Boolean).forEach((userAnswer) => {
-      userAnswer.split("").forEach((char, i) => {
-        if (answer[i] === char) {
-          text += "🟩";
-        } else if (answer.includes(char)) {
-          text += "🟨";
-        } else {
-          text += "⬛";
-        }
-      });
-      text += "%0A";
-    });
-
-    text += "%0Ahttps://katla.vercel.app";
-    const shareToTwitter = `https://twitter.com/intent/tweet?text=${text}`;
+    const text = generateText()
+    const encodeURI = text.replaceAll("\n", "%0A")
```

Comment:
wait, does javascript have a function to encode to URI

Reply:
yes, using `encodeURI` or `encodeURIComponent` I forget which one does what lol

Reply:
but `\n` will become `%5Cn`